src/pages/Dashboard/Markets/SupplyMarket/SupplyMarketTable.tsx

```diff
@@ -91,7 +89,13 @@ export const SupplyMarketTable: React.FC<ISupplyMarketTableUiProps> = ({
       }}
       rowOnClick={rowOnClick}
       rowKeyIndex={0}
-      gridTemplateColumns={styles.getGridTemplateColumns({ isCardLayout: isSmDown })}
+      gridTemplateColumnsCards={styles.getGridTemplateColumns}
+      tableClassName={TABLE_CLASS_NAME}
+      cardClassName={CARD_CLASS_NAME}
+      css={sharedStyles.marketTable({
+        tableClassName: TABLE_CLASS_NAME,
+        cardClassName: CARD_CLASS_NAME,
+      })}
```

Comment:
Since all these classes do is define at what breakpoint each layout should be shown or hidden, why not follow the original idea we had of adding a prop to specify that breakpoint directly?
https://app.clickup.com/t/2gvefed

Reply:
We can. Passing as styles opens up the door to customize more than just the breakpoint to show cards or the table. Eventually, I'd like to break these out into completely separate components that share context for pagination, ordering, filtering ect. In that scenario there would not be a breakpoint prop.